pkg/controller/dynamodb/table/hooks.go

```diff
@@ -83,8 +84,73 @@ func SetupTable(mgr ctrl.Manager, o controller.Options) error {
 			managed.WithConnectionPublishers(cps...)))
 }
 
+// customPointInTimeRecoverySpecification is a custom struct that implements sdk.CustomQuerySerialize
+// we need this to keep aws sdk from dropping the parameter when it's false. we do that by overwriting
+// MarshalQueryParameters for this struct
+type customPointInTimeRecoverySpecification struct {
+	PointInTimeRecoveryEnabled *bool
+}
+
+// MarshalAWS marshals the customPointInTimeRecoverySpecification struct into an AWS request.
+func (c *customPointInTimeRecoverySpecification) MarshalJSON() ([]byte, error) {
+	// If PointInTimeRecoveryEnabled is nil, explicitly set it to false in the request
+	if c.PointInTimeRecoveryEnabled == nil {
+		return []byte(`{"PointInTimeRecoveryEnabled":false}`), nil
+	}
+
+	// Otherwise, marshal the struct as usual
+	return json.Marshal(struct {
+		PointInTimeRecoveryEnabled bool `json:"PointInTimeRecoveryEnabled"`
+	}{
+		PointInTimeRecoveryEnabled: *c.PointInTimeRecoveryEnabled,
+	})
+}
+
+func (c *customPointInTimeRecoverySpecification) ToAWSPointInTimeRecoverySpecification() *svcsdk.PointInTimeRecoverySpecification {
+	if c.PointInTimeRecoveryEnabled == nil {
+		return nil
+	}
+
+	return &svcsdk.PointInTimeRecoverySpecification{
+		PointInTimeRecoveryEnabled: c.PointInTimeRecoveryEnabled,
+	}
+}
+
+func (e *updateClient) postUpdate(_ context.Context, cr *svcapitypes.Table, obj *svcsdk.UpdateTableOutput, _ managed.ExternalUpdate, _ error) (managed.ExternalUpdate, error) {
+	cbresult, err := e.client.DescribeContinuousBackups(&svcsdk.DescribeContinuousBackupsInput{
+		TableName: aws.String(cr.Name),
+	})
+	if err != nil {
+		return managed.ExternalUpdate{}, err
+	}
+	pitrStatus := cbresult.ContinuousBackupsDescription.PointInTimeRecoveryDescription.PointInTimeRecoveryStatus
+	pitrStatusBool := pitrStatusToBool(pitrStatus)
+
+	if !isPitrUpToDate(cr, pitrStatusBool) {
+		pitrSpecEnabled := false
+		if cr.Spec.ForProvider.PointInTimeRecoveryEnabled != nil {
+			pitrSpecEnabled = *cr.Spec.ForProvider.PointInTimeRecoveryEnabled
+		}
```

Comment:
You can use [`k8s.io/utils/pointer`](https://pkg.go.dev/k8s.io/utils/pointer) to simplify this

```suggestion
		pitrSpecEnabled := pointer.BoolDeref(cr.Spec.ForProvider.PointInTimeRecoveryEnabled, false)
```

Reply:
nice, thank you :)

Reply:
nice, thank you! :)